### PR TITLE
fix value in total row in analysis detail section

### PR DESCRIPTION
**Pull request by myarjunar**
### What does it fix?
* Ticket: #4258 
* Funded by: DFAT
* Description:

### CODE REVIEW

**Comment by Gustry** [comment]
ah, you missed the 4.1.1 release.

So the QGIS layer was fine ? only on the reporting side ?

**Comment by myarjunar** [comment]
@Gustry yes the layer is fine

File: safe/report/extractors/analysis_detail.py [hunk 1]
```diff
@@ -361,6 +361,12 @@ def sort_classes(_row):
                 row = details[row_idx]
                 row = row[:column_index] + row[column_index + 1:]
                 details[row_idx] = row
+            # reduce total affected and not affected column index by 1
+            # since we are removing a column
+            if group_key == 'affected':
```
Comment (ismailsunni) [inline safe/report/extractors/analysis_detail.py]: Do we have this `affected` group in the definitions module? If so, it's better to get from the definiton.

**Comment by Charlotte-Morgan** [comment]
@felix-yew  - can you please review this PR

**Comment by timlinux** [comment]
Please ping me when done and I will release as a 4.1.2 bug fix.

**Comment by Charlotte-Morgan** [comment]
+1 for 4.1.2

**Comment by Gustry** [comment]
We need to release 4.1.2 this week. I don't think we will have feedback this week due to holiday.

**Comment by myarjunar** [comment]
would be great of you can check if this issue still appear after I merge this @felix-yew

**Comment by felix-yew** [comment]
:+1: LGTM

**Pull request merged**